android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -859,26 +863,29 @@ private void updateProgressControl(long currentMillis) {
     }
 
     private void setupProgressBarSeekListener() {
-        if (previewSeekBarLayout != null && previewSeekBarLayout.getPreviewView() instanceof ProgressBar) {
-            previewSeekBarLayout.getPreviewView().addOnPreviewChangeListener(new PreviewView.OnPreviewChangeListener() {
-                @Override
-                public void onStartPreview(PreviewView previewView) {
-
-                }
-
-                @Override
-                public void onStopPreview(PreviewView previewView) {
+        if (previewSeekBarLayout != null
+                && previewSeekBarLayout.checkChilds()
+                && previewSeekBarLayout.getPreviewView() instanceof ProgressBar) {
+            if (mPreviewChangeListener == null) {
```

Comment:
I think there was a potential memory leak here.

Reply:
We could have a memory leak here because mPreviewChangeListener is not unregistered.